code/html/index.html

```diff
@@ -1888,6 +1902,10 @@ <h2>
                 <div class="pure-u-1 pure-u-lg-1-4"><label>Pulse time (s)</label></div>
                 <div class="pure-u-1 pure-u-lg-1-4"><input name="relayTime" class="pure-u-1" type="number" min="0" step="0.1" max="3600" /></div>
             </div>
+            <div class="p-g">
+                <div class="p-u-1 p-u-lg-1-4"><label>Nickname</label></div>
+                <div class="p-u-1 p-u-lg-1-4"><input name="relayNickname" class="p-u-1" type="text" original="" /></div>
```

Comment:
original= is a dynamic attr, no need to set it

Reply:
deleted it